Ziro/user.css

```
@@ -504,3 +508,11 @@ input:checked ~ .x-toggle-indicatorWrapper .x-toggle-indicator {
 .XmR5VFpqXOcxRvhuf6OB {
   border-radius: var(--br-1);
 }
+
+.McwcCfBLSuXa5UDU1IMw {
```

```suggestion
.main-rootlist-rootlistDividerContainer {
```
Class has been updated in recent CSS map change